src/test/java/address/unittests/storage/StorageManagerTest.java

```diff
@@ -73,13 +73,70 @@ public void setup(){
     }
 
     @Test
+    public void getConfig_fileInexistent_newConfigCreated() throws Exception {
+        Config dummyConfig = new Config();
+        TestUtil.setFinalStatic(StorageManager.class.getDeclaredField("CONFIG_FILE"), INEXISTENT_FILE.getPath());
+        PowerMockito.whenNew(Config.class).withAnyArguments().thenReturn(dummyConfig);
+        StorageManager.getConfig();
+        PowerMockito.verifyNew(Config.class);
+    }
+
+    @Test
+    public void getConfig_fileExistent_correspondingMethodCalled() throws Exception {
+        FileUtil.createIfMissing(EMPTY_FILE);
+        TestUtil.setFinalStatic(StorageManager.class.getDeclaredField("CONFIG_FILE"), EMPTY_FILE.getPath());
+        StorageManager.getConfig();
+        PowerMockito.verifyPrivate(StorageManager.class).invoke("readFromConfigFile", EMPTY_FILE);
+    }
+
+    @Test
+    public void recreateFile() {} // Requires reflection
```

Comment:
To be clearer, say `This this is not implemented because it requires reflection`.